app/assets/javascripts/Components/dashboard.jsx

```diff
@@ -30,16 +34,66 @@ class Dashboard extends React.Component {
             backgroundColor: 'rgb(75, 192, 192)',
           },
         ],
-      }
+      },
+      gradeEntryFormChart2: {}
```

Comment:
You don't need to create a new state for this: instead, set the `data` state above with the correct data from the server response.

You should, however, create a new state for the Chart *options* (default can be an empty object), and pass that as props to the chart component below.